docs/modules/ROOT/pages/explanation/overview.adoc

```diff
@@ -0,0 +1,54 @@
+= Modelix Overview
+:navtitle: Overview
+
+The goal of this quick overview is for you to understand, what modelix is, what it is not and how it might be utilized.
+
+== What is Modelix?
+
+Modelix is a platform to bring models to the web.
+It consists of multiple components, which can be combined in different ways to support a variety of use cases.
+
+It is important to note, that modelix is neither a language workbench in the traditional sense nor a one-size-fits-all solution.
+The modelix project aims to provide you with building blocks, which empower you to construct your own application with ease.
+
+== Essential components
+
+The following table gives an overview over essential components, which you are very likely to encounter when using modelix.
+For a full list of components check out the xref:reference/components-table.adoc[components table].
+
+|===
+|Component | Short description
+
+|`model-api`
+|Untyped API, which allows to interact with models.
+
+|`model-server`
+|Server, which persistently stores models.
+
+|`model-client`
+|Client, which implements the `model-api` and connects to a `model-server`.
+
+|`model-api-gen(-gradle)`
+|Component, which generates a typed, domain specific API based on a metamodel definition.
+The generated API wraps the untyped `model-api`.
+|===
+
+== Basic example using essential components
+image::example-essential-components.svg[]
+
+The image above shows a basic example of how the essential components work together to form the backend for a web application.
+In this case the metamodel definition is generated from the structure aspect of a language in MPS.
+But it could also come from other sources (symbolized by the question mark).
```

Comment:
if you have to explain the '?', maybe it is not a good symbol for this. alternatively use words: 'your meta-model source'